README.md

```diff
@@ -75,24 +75,29 @@ Unlike openai-whisper, FFmpeg does **not** need to be installed on the system. T
 
 GPU execution requires the following NVIDIA libraries to be installed:
 
-* [cuBLAS for CUDA 11](https://developer.nvidia.com/cublas)
-* [cuDNN 8 for CUDA 11](https://developer.nvidia.com/cudnn)
+* [cuBLAS for CUDA 12](https://developer.nvidia.com/cublas)
+* [cuDNN 8 for CUDA 12](https://developer.nvidia.com/cudnn)
 
-There are multiple ways to install these libraries. The recommended way is described in the official NVIDIA documentation, but we also suggest other installation methods below.
+**Note**: Latest versions of `ctranslate2` support CUDA 12 only. For CUDA 11 support, the current workaround is using a previous version of `faster-whisper` (which depends on a previous version of `ctranslate2`). For more information, see: https://github.com/SYSTRAN/faster-whisper/pull/694, https://github.com/SYSTRAN/faster-whisper/issues/783, https://github.com/SYSTRAN/faster-whisper/issues/717#issuecomment-1963488371
```

Comment:
You don't need to use a previous version of `faster-whisper`, just a previous version of `ctranslate2`.

Reply:
Ah, that's right. Edited!